templates/layout.tpl

```diff
@@ -39,6 +39,12 @@
             </li>
           </ul>
           <ul class="nav navbar-nav navbar-right">
+            <li>
+              <select class="form-control" id="languageSelector">
+                <option value="one">{"Romanian"|_}</option>
+                <option value="two">{"English"|_}</option>
+              </select>
+            </li>
```

Comment:
Dacă ai timp / chef, instalează-ți poedit și fă și traducerile pentru toate constantele string nou introduse. Vezi [instrucțiunile](https://github.com/varena/varena#localization-for-translators).